linux/utils/get_network_config_file.yml

```diff
@@ -0,0 +1,74 @@
+# Copyright 2023 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Get network network config file for an interface
+# Parameters:
+#   network_adapter_name: The network interface name
+# Return:
+#   network_config_path: The network interface config file path
+#
+- name: "Initialize network config file for {{ network_adapter_name }}"
+  ansible.builtin.set_fact:
+    network_config_path: ""
+
+- name: "Get network device manager in guest OS"
+  include_tasks: get_network_manager.yml
+  when: guest_os_network_manager is undefined or not guest_os_network_manager
+
+- name: "Get network config file from NetworkManager"
+  block:
+    - name: "Get network connections"
+      ansible.builtin.shell: "nmcli -t -f NAME,ACTIVE,FILENAME connection show --active | grep '^{{ network_adapter_name }}:'"
+      delegate_to: "{{ vm_guest_ip }}"
+      register: "network_conn_result"
+      ignore_errors: true
+
+    - name: "Set fact of network config file for '{{ network_adapter_name }}' on {{ guest_os_ansible_distribution }}"
+      ansible.builtin.set_fact:
+        network_config_path: "{{ network_conn_result.stdout_lines[0].split(':')[2] }}"
+      when:
+        - network_conn_result.stdout_lines | length == 1
```

Comment:
Is here need to add "not network_conn_result.failed" or "network_conn_result.rc ==0" to make sure command executed without error then to get the result?

Reply:
Fixed, thanks!